src/CodeGeneration.Roslyn.Tests/CodeGeneration.Roslyn.Tests.csproj

```diff
@@ -7,6 +7,7 @@
     <GenerateCodeFromAttributesToolPathOverride>$(OutputPath)..\..\CodeGeneration.Roslyn.Tool\$(Configuration)\netcoreapp2.1\dotnet-codegen.dll</GenerateCodeFromAttributesToolPathOverride>
     <!-- Ignore CGR1002 because it's expected to be raised by legacy Amadevus.RecordGenerator -->
     <MSBuildWarningsAsMessages>$(MSBuildWarningsAsMessages);CGR1002</MSBuildWarningsAsMessages>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Can we maybe move this up to `Directory.Build.props`? I think it shouldn't add any warnings.

Reply:
> Can we maybe move this up to Directory.Build.props? I think it shouldn't add any warnings.
I'll do. It has 0 warnings afterwards.

Reply:
The change adds some messages, stating to use some C# 8.0 features. `IDE0063 'using' statement can be simplified`, ...